dvc/path/local.py

```diff
@@ -8,4 +8,9 @@ class PathLOCAL(PathBASE):
     scheme = Schemes.LOCAL
 
     def __str__(self):
-        return os.path.relpath(self.path)
+        try:
+            return os.path.relpath(self.path)
+        except ValueError:
```

Comment:
Hm, still catching ValueError here seems too much, since it is only relevant for windows and ValueError could be raised in a lot of cases. Maybe it is worth checking for it separately before trying to relpath()?

Reply:
Btw, just a note: that check and relpath itself might cause quite a performance hit, as we've learned from cProfile a long time ago, so the plan was to make PathLOCAL do some caching in the future, so it doesn't have to recompute that again and again.